src/components/DetailsPostcodeHint.svelte

```diff
@@ -53,20 +53,31 @@
           aria-label="Close"
           onclick={closeHint}
   ></button>
-  <p>
-    Nightly calculated from nearby places having this postcode.
-    <a href="https://nominatim.org/release-docs/latest/admin/Maintenance/#updating-postcodes">
-      Documentation
-    </a>.
-  </p>
-  <p>
-    You can search for those with an
-    <a href={url} target="_blank" rel="noreferrer">Overpass Turbo query</a>.
-  </p>
-  <p>
-    <a href="https://nominatim.org/2022/06/26/state-of-postcodes.html"
-       target="_blank" rel="noreferrer">How Nominatim uses postcodes</a>.
-  </p>
+  {#if country_code === 'gb'}
+    <p>
+      Based on an imported dataset based on Royalmail data.
```

Comment:
UK postcode data is also only supplemented by the external data. If anything is available in OSM, it takes precedence.

Also, I understood that the UK source is open data by OS, not Royal Mail?